environment_acceptance.sh

```diff
@@ -1,3 +1,5 @@
 export SETTINGS='config.TestConfig'
 export PPI_END_POINT='http://landregistry.local:8002/get_ppi'
-export ELASTIC_SEARCH_ENDPOINT='http://ec2-54-72-123-155.eu-west-1.compute.amazonaws.com::4300'
+export ELASTIC_SEARCH_ENDPOINT='http://ec2-54-77-238-128.eu-west-1.compute.amazonaws.com:4300'
```

Comment:
why are we changing back the elastic search endpoint here? :) believe we want to stay on 123 155, but without two "dots" in the URL where the port is? (cc @nickgravgaard is that the bug you were hunting?)